src/SubmitButton/Styles.css

```diff
@@ -0,0 +1,6 @@
+
+.defaultButton{
+    display: flex;
```

Comment:
Please change name of the class so it's consistent with name of the component (but start with small key)